pkg/api/v1/types.go

```diff
@@ -2503,6 +2503,8 @@ type EventSource struct {
 	Component string `json:"component,omitempty"`
 	// Host name on which the event is generated.
 	Host string `json:"host,omitempty"`
+	// Scheduler name on which the event is generated.
+	SchedulerName string `json:"schedulerName,omitempty"`
```

Comment:
Thanks for detailed reasons. But I have a question, If component only include the scheduler name, and  the scheduler name is like aaa, or bbb, etc, we couldn't know events is from a scheduler. If I want to know whether events is from a scheduler, how could I do? Make a strict rule that the scheduler name should be prefixed with scheduler, like scheduler-aaa-01, scheduler-aaa-02?